src/main/java/io/codelex/flowcontrol/exercises/Exercise_5.java

```diff
@@ -0,0 +1,32 @@
+package io.codelex.flowcontrol.exercises;
+
+import java.util.Scanner;
+
+public class Exercise_5 {
+    public static void main(String[] args) {
+        phoneKeyPad();
+    }
+
+    public static void phoneKeyPad() {
```

Comment:
ja metodi domāts izmantot tikai taipašā klasē, tad jāizmanto private, citur tad to vairs nepieminešu